app/views/help/cookie_settings.html.erb

```diff
@@ -1,6 +1,20 @@
 <% content_for :title, "Cookies on GOV.UK" %>
 
 <main id="content" role="main" class="group help-page cookie-settings">
+
```

Comment:
Need to add aria-hidden when appropriate.

Reply:
This doesn't seem to be an issue - so, bearing in mind the [first rule of ARIA](https://www.w3.org/TR/using-aria/#rule1), I've held off from adding it. I'll dig into this a bit deeper to, but I don't want this to be a blocker.